src/app/(home)/globe/index.tsx

```diff
@@ -113,7 +157,16 @@ function GlobeInner() {
               type="fill"
               source-layer="default"
               paint={{
-                "fill-color": "rgba(211, 211, 211, 0.3)", // Transparent blue
+                "fill-color": "rgba(250, 250, 249, 0.3)", // Transparent blue
+              }}
+            />
+            <Layer
+              id="area-hovered-polygon"
+              type="fill"
+              source-layer="default"
+              filter={highlightFilter as any} // eslint-disable-line @typescript-eslint/no-explicit-any
+              paint={{
+                "fill-color": "rgba(250, 250, 249, 0.7)", // Transparent blue
               }}
```

Comment:
@oliverroick the layer is rendering all the areas that the mouse has hovered over. If you move the pointer fast enough, it will take a while to catch up. We might need to throttle the re-rendering to avoid this situation.